src/components/ButtonPanel/ButtonPanel.js

```diff
@@ -6,38 +6,43 @@ const buttonPanel = ({ clickHandler }) => {
 
   return (
     <div>
-      <div className = "group group-1">
+
+    // <div className = "button" style = {style} onClick ={handleClick}>
+    //   <h3>{props.name}</h3>
+    // </div>
+
+      <div className = "group group-1 ">
```

Comment:
Please create your keys/buttons dynamically by using two for-loops. The first one will iterate over the groups you have and the second one will iterate over the buttons in the particular group.